sql/moz-fx-data-shared-prod/fx_quant_user_research/dataset_metadata.yaml

```diff
@@ -4,13 +4,10 @@ description: |-
 dataset_base_acl: view
 user_facing: true
 labels: {}
-default_table_workgroup_access:
-- role: roles/bigquery.dataViewer
-  members:
-  - workgroup:mozilla-confidential
-default_table_expiration_ms: null
 workgroup_access:
 - role: roles/bigquery.dataViewer
   members:
   - workgroup:mozilla-confidential
+  - workgroup:fx-quant-user-research/data-developers
+  - workgroup:fx-quant-user-research/data-viewers
```

Comment:
Since `workgroup:mozilla-confidential` is effectively everyone at Mozilla, also granting access to these other specific working groups doesn't really seem necessary.  Or do you want this data to _only_ be accessible to those more specific workgroups?

Reply:
I wouldn't go so far as to consider this an anti-pattern, I mainly just wanted to make sure you weren't expecting access to be restricted when everyone at Mozilla will have access as it is.